src/platform/web/ui/general/LoadingView.ts

```diff
@@ -17,7 +17,8 @@ limitations under the License.
 import {TemplateView} from "./TemplateView";
 import {spinner} from "../common.js";
 
-export class LoadingView extends TemplateView {
+export class LoadingView <T> extends TemplateView <T> {
```

Comment:
`T` is the type of the view model here, so you can use `any` as the type.
```suggestion
export class LoadingView extends TemplateView<any> {
```